doc(release) Release notes cloud 25.03 (#4320)
centreon/centreon-documentation: cloud/releases/cloud-release-notes.md

title: Centreon Cloud release notes
---

## June 12th, 2025

### Centreon IT Edition

**Centreon Monitoring Agent Beta**

The Beta version of the Centreon Monitoring Agent is now available for our Centreon Cloud customers.
The Centreon Monitoring Agent (CMA) collects metrics and computes statuses on the servers it monitors, and sends them to Centreon. 
Centreon plugins as well as Nagios-compatible plugins can be used with this agent.
More information about configuration is available in our [offical documentation](/pp/integrations/plugin-packs/getting-started/how-to-guides/cma/) and in a dedicated [TheWatch group](https://thewatch.centreon.com/groups/opentelemetry-agent-beta-program-61).

**Centreon Dashboards**

> No big new feature for our Centreon Dashboards feature this time but still few improvements have been made.

Reviewer: ```suggestion
No big new features for Centreon Dashboards this time, but still some improvements have been made.
```

> Readability of Business Activities Diagram widget has been improved by adding host name to service KPIs and increasing the number of characters displayed. 

Reviewer: ```suggestion
The readability of the Business Activities Diagram widget has been improved by adding the host's name to service KPIs and by increasing the number of characters displayed. 
```

> Designing dashboards hes been eased by the possibility the possibility to resize the widgets in all direction, and reduce unused spacing between widgets.

Reviewer: ```suggestion
Designing dashboards has been made easier: you can now resize the widgets in all directions, and reduce unused spacing between widgets.
```

> **Resource stattus**

Reviewer: ```suggestion
**Resource status**
```

> Resource Status page performances have been improved by optimizing filtering requests.

Reviewer: ```suggestion
Performance for the Resource Status page has been improved by optimizing filtering requests.
```

### Centreon Business Edition

**MAP**

> We are enhancing access control on maps by adding the possibility to assign privileges amongst owner, editor, viewer or none to Geo Views.

Reviewer: ```suggestion
We have enhanced access control on maps: you can now assign privileges (owner, editor, viewer or none) to geographic views.
```

> Note that for now, it is only possible to select ACL groups derived either from roles (Administrator, Editor, Viewer) or from Resource Access Management feature. Section based on user groups will follow.

Reviewer: ```suggestion
Note that for now, it is only possible to select ACL groups derived either from roles (Administrator, Editor, Viewer) or from Resource Access Management rules. A section based on user groups will follow.
```

> Widget to dispaly output of a check has also been improved by adding more macros editor can display.

Reviewer: ```suggestion
The widget that displays the output of a check has also been improved by adding more macros that the editor can display.
```
